### PR TITLE
Prerelease of v1.0.0 rc.2

### PR DESCRIPTION
**Pull request by flavens**

### CODE REVIEW

File: projects/dsp-ui/package.json [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+  "keywords": [
+    "dasch service platform",
+    "user interface",
+    "angular",
+    "knora",
+    "humanities"
+  ],
   "author": "Data and Service Center for the Humanities DaSCH, University of Basel",
-  "license" : "AGPL-3.0-or-later",
+  "license": "AGPL-3.0-or-later",
```
Comment (kilchenmann) [inline projects/dsp-ui/package.json:18]: This looks great. Can you do the same in main `package.json`?

Comment (flavens) [reply projects/dsp-ui/package.json:18]: what is different in `package.json`? I cannot see any

Comment (flavens) [reply projects/dsp-ui/package.json:18]: ah keywords maybe?

Comment (kilchenmann) [reply projects/dsp-ui/package.json:18]: This was nonsense. I'm sorry

**Pull request merged**